include/Projektmanagementplan.tex

```diff
@@ -48,6 +48,23 @@ \subsection{Grobplan}
+
+\subsection{Sprint 03}
+Aus der Meilensteinsitzung wurde klar, dass die Kosten für die ermittelten Geräte für Testzwecke und Prototypisierung zu hoch sind, und die nötigen Geldmittel weder vom Kunden noch vonseiten der Hochschule zur Verfügung stehen. Als Sprintziel wurde die Akquisition von Versuchsgeräten definiert. Das Projektteam versuchte noch einmal Testgeräte bei Bildungseinrichtungen, Hochschulen und Herstellern zu erhalten. Herr Baumann kontaktierte dafür Gewerbeschulen in der Zentralschweiz und Kompetenzzentren für RFID an den Hochschulen Luzern und Winterthur. Herr Wicki kontaktierte Vertreiber von HF RFID Lösungen, ob diese Leihgeräte zur Verfügung stellen könnten. Weiter wurden auch die Budgetvorstellungen der Hochschule dem Projektteam mitgeteilt. Weiter wurde vom Projektteam das Protokoll für die Meilensteinsitzung erarbeitet. Die Herausforderung in diesem Sprint waren vor allem die Schwierigkeiten im Bezug auf die Versuchsgeräte.
+
+\subsection{Sprint 04}
+In diesem Sprint wurden die erarbeiteten Konzepte noch einmal kritisch begutachtet, harmonisiert und für die Machbarkeitsstudie finalisiert. Alle Versuche Leihgaben zu erhalten scheiterten jedoch. Vonseiten der Hersteller erhielt man zwar Rabatte. Die Preise für die Geräte waren jedoch immer noch ein Vielfaches von dem, was die Hochschule bereit war für eine BDA zur Verfügung zu stellen. Das Sprintziel lag demnach darin, einen Grundlagenentscheid im Bezug auf die Versuche zu erreichen. Das Team entschied sich, um reelle Aussagen in der Machbarkeitsstudie machen zu können, auf eigene Kosten bei einem Hersteller in China zu bestellen. Diese Geräte trafen dann auf Ende Sprint beim Team ein. Um nicht zuviel Zeit zu verlieren, entschied das Team sich, Versuche genau zu spezifizieren, Versuchsmaterialien zu beschaffen und ein Testframework zu definieren. Am Ende dieses Sprints stand die Zwischenpräsentation des Projekts an. Diese wurde von beiden Teammitgliedern erarbeitet, antrainiert und dem Betreuer und Experten präsentiert. Die Herausforderung dieses Sprints war der Motivationsverlust aufgrund der Schwierigkeiten und Ungewissheit im Bezug auf die Versuchsgeräte und dem weiteren Verlauf des Projekts.
```

Comment:
Problem aus Sprint 03 wurde mitgenommen, als Ziel galt es dieses Problem zu lösen. Get a RFID reader as cheap as possible and it has to arrive at the end of this sprint.

Reply:
ein Testframework zu definieren -> und für die durchführung der Test eine Applikation zu erstellen ..... (irgenwie noch sagen, dass es erstellt werden soll, damit das protokoll genauer sein kann sowie die Zeitkritischen funktionen auf die ms genau gemesen werden können, was von hand gar nicht oder nur begrenz möglich sei)